schema/tables/package_install_history.yml

```
@@ -1,7 +1,25 @@
 name: package_install_history
+description: The Fleet `package_install_history` table provides a detailed log of all installed packages on macOS.
```

Can you change the description of this table?

```suggestion
description: The `package_install_history` osquery table provides a detailed log of all installed packages on macOS.
```